domains/transfer/Shared/ProjectOriginClients.Tests/ProjectOriginWalletClientTests.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using ProjectOriginClients.Models;
+using ProjectOriginClients.Tests.Testcontainers;
+using Xunit;
+
+namespace ProjectOriginClients.Tests;
+
+public class ProjectOriginWalletClientTests : IClassFixture<ProjectOriginStack>
```

Comment:
Add ProjectOriginStack as CollectionFixture in IntegrationTestCollection and add this class as part of the collection.

See: https://github.com/Energinet-DataHub/energy-origin/blob/main/domains/certificates/Query.API/API.IntegrationTests/IntegrationTestCollection.cs

Reply:
It's only used for ProjectOriginClientTests, so I think it's ok to keep them separate?

Reply:
I dont agree, this is exactly why we ended up with more of them in Certificates.